vhdbuilder/packer/install-dependencies.sh

```diff
@@ -458,19 +469,25 @@ PATCHED_HYPERKUBE_IMAGES="
 1.15.7-hotfix.20200326
 1.15.10-hotfix.20200408.1
 1.15.11-hotfix.20200529.1
+1.15.12-hotfix.20200623.1
 1.16.7-hotfix.20200601.1
 1.16.8.1
 1.16.9-hotfix.20200529.1
+1.16.10-hotfix.20200623.1
 1.17.3-hotfix.20200601.1
 1.17.4.1
 1.17.5.1
+1.17.7-hotfix.20200624
 1.18.1.1
 1.18.2.1
```

Comment:
should we remove 1.18.2.1?

Reply:
not until the toggle for 1.18.2-hotfix.20200624 is fully out - otherwise some vmas scale will still try to download 1.18.2.1.